.tina/collections/company.tsx

```diff
@@ -51,6 +54,43 @@ export const companySchema: Collection = {
       label: "Show Regional Director Panel",
       required: false,
     },
+    {
+      type: "boolean",
+      name: "showTestimonials",
+      label: "Show Testimonials",
+    },
+    {
+      type: "boolean",
+      name: "showAllTestimonials",
+      label: "Show All Testimonials",
+    },
+    {
+      type: "boolean",
+      name: "HideInternshipTestimonials",
+      label: "Hide Internship Testimonials from All",
+    },
+    {
+      type: "object",
+      label: "Testimonial Categories",
+      name: "testimonialCategories",
+      ui: {
+        itemProps(item) {
+          return {
+            label:
+              item.testimonialCategory ?? "Select your testimonial category",
+          };
+        },
+      },
+      list: true,
+      fields: [
+        {
+          type: "reference",
+          label: "Testimonial Category",
+          name: "testimonialCategory",
+          collections: ["testimonialCategories"],
+        },
+      ],
+    },
```

Comment:
Can we instead move these fields to `TestimonialList`, so these aren't global fields for every company page?